src/tests/unit/inference_engine/query_model_test.cpp

```diff
@@ -0,0 +1,307 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <gtest/gtest.h>
+#include <iostream>
+
+#include <openvino/pass/manager.hpp>
+#include <ngraph/opsets/opset9.hpp>
```

Comment:
Can we use only new API?
```suggestion
#include <openvino/opsets/opset9.hpp>
```
Please remove ngraph from new tests